src/actions.js

```diff
@@ -45,14 +45,26 @@ export const fetchItems = (url, dispatch) => {
           reject(new Error('Request timed out'));
         }, FETCH_TIMEOUT);
 
-        fetch(url, {
+        window.fetch(url, {
           credentials: 'include',
           headers: {
             Accept: 'application/json'
           }
         })
-          .json
-          .then(resolve, reject);
+          .then(response => {
+            return response.json()
+              .then(json => {
+                if (response.status > 399) {
+                  const err = new Error(json.message || DEFAULT_ERROR_MESSAGE);
```

Comment:
I'd actually make the default message here _a little bit_ specific and useful.

`Fetch failed with status code ${response.status}` for example.

Reply:
Then we can get a more useful message in dev, and the error message genericiser in your other PR will revert it to a default message down the pipe.